auslib/db.py

```diff
@@ -28,19 +28,14 @@
 import logging
 
 
-def rowsToDicts(fn):
-    """Decorator that converts the result of any function returning a dict-like
-       object to an actual dict. Eg, converts read-only row objects to writable
-       dicts."""
-    def convertRows(*args, **kwargs):
-        ret = []
-        for row in fn(*args, **kwargs):
-            d = {}
-            for key in row.keys():
-                d[key] = row[key]
-            ret.append(d)
-        return ret
-    return convertRows
+def rows_to_dicts(rows):
+    """Converts SQL Alchemy result rows to dicts.
+
+    You might want this if you want to mutate objects (SQLAlchemy rows
+    are immutable), or if you want to serialize them to JSON
+    (SQLAlchemy rows get confused if you try to serialize them).
+    """
+    return map(dict, rows)
```

Comment:
Thank you for reworking this. The decorator seemed like a good idea when it was first written, but it's grown very long in the tooth.